api.js

```diff
@@ -1,9 +1,17 @@
 var mongo = require('mongodb').MongoClient,
-    fiddles = null;
+    fiddles = null,
+    mongoConnectionStr = process.env.MONGOHQ_URL ?
```

Comment:
To me, if you didn't take time to update your environment then defaulting to an empty string will end up with the same result (not connecting to a db).